Qsi/Engines/Explain/Analyzers/ExplainActionAnalyzer.cs

```diff
@@ -70,11 +70,15 @@ private QsiExplainDataManipulationResult ConvertToExplain(QsiDataManipulationRes
                     operation |= QsiDataValueOperation.Delete;
             }
 
-            return new QsiExplainDataManipulationResult(
+            var explainResult = new QsiExplainDataManipulationResult(
                 result.Table,
                 result.AffectedColumns,
                 operations
             );
+
+            explainResult.SensitiveDataCollection.AddRange(result.SensitiveDataCollection);
+
+            return explainResult;
```

Comment:
[May]
`SesnsitiveDataCollection`에 init를 추가하고, 다음과 같은 형태로 사용할 수 있습니다.
```suggestion
            return new QsiExplainDataManipulationResult(result.Table, result.AffectedColumns, operations)
            {
            	SensitiveDataCollection = result.SensitiveDataCollection
            };
```

Reply:
넵 추후에 해당 부분 수정이 있을때 검토해보겠습니다.